client/registry/github_registry.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package registry
+
+import (
+	"github.com/google/go-github/github"
+
+	"fmt"
+	"log"
+)
+
+// GithubRegistry implements the Registry interface that talks to github.
+type GithubRegistry struct {
+	owner      string
```

Comment:
Actually the github API calls it owner :)
https://github.com/google/go-github/blob/master/github/repos_contents.go#L130